selection_sort/Python/selection_sort.py

```diff
@@ -0,0 +1,17 @@
+def selection_sort(arr):
+    for i in range(len(arr)):
+        min_i = min(arr[i:]) 
+        min_index = arr[i:].index(min_i) 
+        arr[i + min_index] = arr[i] 
+        arr[i] = min_i
+    sorted_arr = arr
+    return sorted_arr
+
+def main():
+    arr = [2, 3, 0, 4]
+    sorted_arr = selection_sort(arr)
+    print(sorted_arr)
+
+    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
+++ b/tmp/tmpxipfo4mc/selection_sort/Python/selection_sort.py
@@ -12,6 +12,6 @@
     sorted_arr = selection_sort(arr)
     print(sorted_arr)
 
-    
+
 if __name__ == '__main__':
     main()
```